gen3-client/g3cmd/root.go

```diff
@@ -35,7 +35,7 @@ func init() {
 	cobra.OnInitialize(initConfig)
 
 	// Define flags and configuration settings.
-	RootCmd.PersistentFlags().StringVar(&profile, "profile", "default", "Specify profile to add or edit with --profile=<profile-name>")
+	RootCmd.PersistentFlags().StringVar(&profile, "profile", "", "Specify profile to add or edit with --profile=<profile-name>")
```

Comment:
since we can't make `profile` required globally, could you make it required for individual commands in the same way it's required for the `auth` command?

Reply:
fixed in https://github.com/uc-cdis/cdis-data-client/pull/59/commits/72811b967306e1963a3708299220ba7ce59803c9